tpl/page_optm/settings_media.tpl.php

```diff
@@ -70,8 +70,8 @@
 		<td>
 			<?php $this->build_input( $id, 'litespeed-input-long' ); ?>
 			<div class="litespeed-desc">
-				<?php echo __( 'Specify an SVG to be used as a placeholder when generating locally.', 'litespeed-cache' ); ?>
-				<?php echo __( 'It will be converted to a base64 placeholder on-the-fly.', 'litespeed-cache' ); ?>
+				<?php echo __( 'Specify a SVG to be used as a placeholder when generating locally.', 'litespeed-cache' ); ?>
```

Comment:
I say "SVG" as the letters S V and G, so "an" would be appropriate here.

Is there a more standard way to say SVG that would make "a" a better choice?

Reply:
No, I was thinking that since it started with a consonant but after looking it up then its more on how it sounds like & "es" is more how we pronounce it so you're right.